### DISCUSSION TITLE
有没有做生成TS类型Interface的代码或者文件的计划。

### DISCUSSION THREAD

**Discussion by adoin** [xiaoymin/knife4j#634] [Ideas]
RT

**Comment by xiaoymin** [comment]
早就有了吧，不知道你说的是那种https://doc.xiaominfo.com/demo/doc.html

![image](https://github.com/xiaoymin/knife4j/assets/7894406/785160f5-22c8-4345-acb6-1f2bd187d0ff)

**Reply by adoin** [reply]
是带naming的。 比如是输出User类相关的数据 自动能成Result<User>  然后interface User{id?:number}类似

**Reply by xiaoymin** [reply]
上面这个图中不是interface class么？

**Reply by adoin** [reply]
实际中不太会是 param1  res1这样吧。 我还没升级knife4j。 现在已经具备命名能力了吗
